client/src/main/java/de/hellfish/ngrok/client/ClientRunner.java

```diff
@@ -0,0 +1,71 @@
+package de.hellfish.ngrok.client;
+
+import de.hellfish.ngrok.utils.HttpRequestSerializer;
+import de.hellfish.ngrok.utils.MyHttpRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+
+
+@Slf4j
+@Component
+public class ClientRunner implements CommandLineRunner {
+    private static int servicePort = 2222;
+    private static final String serverHost = "localhost";
+    private static final int serverPort = 8082;
+    private static boolean running = true;
```

Comment:
This variable is not needed. Just throw an exception from `initConnectionWithServer` and on 38 line just put `return`.